apps/survey/src/util/gtmEvent.ts

```diff
@@ -0,0 +1,22 @@
+import { useGtm } from '@gtm-support/vue-gtm';
+import { useSurvey } from '@intake24/survey/stores';
+import type { GtmEventParams } from '@intake24/ui/tracking';
+
+export function sendGtmEvent(params: GtmEventParams): void {
+  const eventParams = { ...params };
+  eventParams.noninteraction = false;
+  try {
+    const survey = useSurvey();
+    if (survey) {
```

Comment:
if check should not be needed, I'd refactor to something like this to be clear what overrides what.

```ts
try {
    const survey = useSurvey();

    const defaults = {
      noninteraction: false,
      uxSessionId: survey.data.uxSessionId,
      uxUserId: survey.user?.userId,
    };

    useGtm()?.trackEvent({ ...defaults, ...params });
  }
  catch (e) {
    console.error('Error sending GTM event:', e);
  }
```